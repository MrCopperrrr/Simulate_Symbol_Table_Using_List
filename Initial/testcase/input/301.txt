INSERT x number
PRINT
BEGIN
INSERT y string
PRINT
BEGIN
INSERT z string
PRINT
END
INSERT w number
PRINT
END
INSERT v string
PRINT