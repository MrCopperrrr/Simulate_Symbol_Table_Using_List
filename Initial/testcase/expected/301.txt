success
x//0
success
x//0 y//1
success
x//0 y//1 z//2
success
x//0 y//1 w//1
success
x//0 v//0